app/controllers/answers_controller.rb

```diff
@@ -37,6 +37,11 @@ def destroy
     end
   end
 
+  def accept
```

Comment:
I have done that boss. I only added the `:accept` to the array of the before actions in the `OwnershipConcern`. So cool is reusable code. :recycle: